apps/docs-next/src/content/learn/getting-started/your-first-scene.mdx

```diff
@@ -109,143 +60,143 @@ the underlying Three.js object and attaches it to the correct property.
+    pattern so Threlte will take care of it. It checks for the properties `isMaterial` and `isGeometry` on
+    the underlying Three.js object and attaches it to the correct property.
+  </details>
+</Tip>
 
 <details>
 	<summary>
 		Three.js equivalent
 	</summary>
-
-```ts
-const geometry = new THREE.BoxGeometry(1, 2, 1)
-```
-
+  ```ts
+  // creating the objects
+  const geometry = new THREE.BoxGeometry()
+  const material = new THREE.MeshBasicMaterial()
+  const mesh = new THREE.Mesh()
+
+  // "attaching" the objects
+  mesh.geometry = geometry
+  mesh.material = material
+  ```
 </details>
 
-## Adding Props
+## Modifying Objects
+
 
-That cube is still a bit boring. Let's add some color to it! We can do this by passing props to the
-`<T>` component. In this case we're passing a `color` prop to the `<T.MeshBasicMaterial>` component.
+That cube is still a bit boring. Let's add some color to it, and make it a bit bigger! We also want to move it up a little to highlight it. We can do this by passing props to the
+`<T>` component.
 
 {/* prettier-ignore-start */}
-```svelte title="Scene.svelte" {7}m
+```svelte title="Scene.svelte" {5-7}m
 <script>
   import { T } from '@threlte/core'
 </script>
 
-<T.Mesh>
+<T.Mesh position.y={1}>
   <T.BoxGeometry args={[1, 2, 1]} />
   <T.MeshBasicMaterial color="hotpink" />
 </T.Mesh>
 ```
 {/* prettier-ignore-end */}
 
-<details>
-	<summary>
-		Three.js equivalent
-	</summary>
-
-```ts
-const material = new THREE.MeshBasicMaterial()
-material.color.set('hotpink')
-```
-
-</details>
 
-Threlte has some rules of what we can pass as props to `<T>` components. First, it will try to find
-the **property on the underlying Three.js object** based on the **name of the prop**. In this case it's the property [`color` on our `THREE.MeshBasicMaterial`](https://threejs.org/docs/index.html#api/en/materials/MeshBasicMaterial.color).
-
-It will then look for a method `set` on that property and use it to set the new value. In our example
-it will call `material.color.set('hotpink')` to set the color of our material.
-
-If there's no `set` method, it will try to set the property directly.
+<Example
+  path="first-scene/step-2"
+  hideCode
+/>
+  
+Threlte automatically generates props for `<T>` based on the underlying Three.js object. This means you can easily guess most `<T>` props 
+based on the <a href="https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene">Three.js docs</a> for the class you are using.
 
-If there's a `set` method and it accepts more than one value (such as a `THREE.Vector3`: `vec3.set(1, 2, 3)`), we can pass an array as a prop. Let's use this to set the position of our cube.
+<Tip type="info">
+  #### Props
+  
+  The special **args** prop we use in `<T.BoxGeometry>` corresponds to the object's constructor arguments. Other so-called **auto props** - like `color` in our `<T.MeshBasicMaterial>`.
```

Comment:
"Props interpreted from the underlying Three.js object are called "Auto Props", like …"